ui/page/home/view.jsx

```diff
@@ -204,20 +213,7 @@ function HomePage(props: Props) {
 
   return (
     <Page className="homePage-wrapper" fullWidthPage>
-      <Meme meme={homepageMeme} />
-
-      {!fetchingActiveLivestreams && (
-        <>
-          {authenticated && subscriptionChannelIds.length > 0 && !hideScheduledLivestreams && (
-            <ScheduledStreams
-              channelIds={subscriptionChannelIds}
-              tileLayout
-              liveUris={getLivestreamUris(activeLivestreams, subscriptionChannelIds)}
-              limitClaimsPerChannel={2}
-            />
-          )}
-        </>
-      )}
+      {/* <Meme meme={homepageMeme} /> */}
```

Comment:
The meme headline will be discontinued with these changes? R.I.P.

Reply:
It's back when the top entry is neither a banner nor portals.